heureka/ui/src/App.js

```diff
@@ -27,6 +34,33 @@ const App = (props) => {
   w-full
   `
 
+  useLayoutEffect(() => {
+    // filterLabels are the labels shown in the filter dropdown, enabling users to filter services based on specific criteria. Default is support-group.
+    if (props.filterLabels) setLabels(props.filterLabels)
+
+    // predefined filters config
+    const predefinedFilters = [
+      {
+        name: "support-group",
+        displayName: "support-group",
+        // regex that matches all entities with default filter
+        region: "\bcontainers\b",
+      },
+    ]
+    // setPredefinedFilters(predefinedFilters)
+
+    // initially active predefined filter
+    const initialPredefinedFilter = "support-group"
+    // setActivePredefinedFilter(initialPredefinedFilter)
+
+    // save the apiEndpoint. It is also used outside the alertManager hook
+    setApiEndpoint(props.endpoint)
+  }, [])
+
+  useLayoutEffect(() => {
```

Comment:
again the layout effect question. Does this have to be a layouteffect?

Reply:
I think so, as I explained in the other section.